styles.css

```diff
@@ -336,3 +334,101 @@ body {
 .skills h4 {
   font-size: 20px;
 }
+
+/* Contact Form */
+
+.contact-forms {
+  font-family: Poppins, sans-serif;
+  background-image: url("images/Contact-me.svg");
+  background-repeat: no-repeat;
+  min-height: 100vh;
+}
+
+.contact {
+  margin-top: 103px;
+  text-align: center;
+}
+
+.form-heading {
+  color: #fff;
+  font-style: bold;
+}
+
+.form-paragraph {
+  margin-top: 12px;
+  margin-left: 24px;
+  margin-right: 24px;
+  text-align: center;
+  font-size: 20px;
+  color: #fff;
+}
+
+.name {
+  margin-left: 24px;
+  margin-right: 24px;
```

Comment:
- [ ] Please add the correct margin-top for the form heading and fix the right margin for the whole form section and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/161133781-b9f16e4f-c893-42cd-9690-48030e342543.png) | ![image](https://user-images.githubusercontent.com/68796232/161133966-eae37242-c1b3-463f-8a43-eb803c916723.png)  |